x/tally/types/filters.go

```diff
@@ -126,13 +129,15 @@ func NewFilterStdDev(input []byte, gasCostMultiplier uint64, replicationFactor u
 
 	switch input[9] {
 	case 0x00: // Int32
-		filter.filterFunc = detectOutliersSignedInteger[int32]
+		filter.filterFunc = detectOutliersInt[int32]
 	case 0x01: // Int64
-		filter.filterFunc = detectOutliersSignedInteger[int64]
+		filter.filterFunc = detectOutliersInt[int64]
 	case 0x02: // Uint32
-		filter.filterFunc = detectOutliersUnsignedInt[uint32]
+		filter.filterFunc = detectOutliersInt[uint32]
 	case 0x03: // Uint64
-		filter.filterFunc = detectOutliersUnsignedInt[uint64]
+		filter.filterFunc = detectOutliersInt[uint64]
+	case 0x04: // BigInt
```

Comment:
For completeness I think we should add the `(u)int128` and `(u)int256` here as well, even though it all ends up in the same bucket. Makes it easier if at some point we do distinguish between the types.

Reply:
Added four different input types based on bit length of the big Int encoding: 32, 64, 128, and 256.